packages/components/psammead-media-player/src/Placeholder/index.jsx

```diff
@@ -12,15 +14,57 @@ const StyledPlaceholder = styled.div`
   height: 100%;
 `;
 
-const Placeholder = ({ onClick, src }) => (
-  <StyledPlaceholder onClick={onClick}>
-    <Image alt="Image Alt" src={src} />
-  </StyledPlaceholder>
-);
+const StyledPlayButton = styled(PlayButton)`
+  position: absolute;
+  bottom: 0;
+
+  /* stylelint-disable */
+  /* https://www.styled-components.com/docs/advanced#referring-to-other-components */
+  ${StyledPlaceholder}:hover &,
+  ${StyledPlaceholder}:focus & {
+    background-color: ${C_POSTBOX};
+  }
+  /* stylelint-enable */
+`;
+
+const Placeholder = ({ onClick, service, src, mediaInfo }) => {
+  const { title, datetime, duration, durationSpoken, type } = mediaInfo;
+  return (
+    <StyledPlaceholder onClick={() => {}}>
```

Comment:
Is this meant to be a no-op? Both the placeholder (and play button) should trigger playback. Incidentally this is why hovering the placeholder triggers a hover state on the play button.